umbraco/Umbraco-CMS
# v9: Added missing namespaces to viewimports

fixes https://github.com/umbraco/Umbraco-CMS/issues/10776 

# Notes 
- Added Umbraco.Cms.Core.Models.PublishedContent namespace to _Viewimports, as the Fallback class uses this namespace and Fallback is used commonly
- Added Microsoft.AspNetCore.Html to _ViewImports, as the HtmlString class uses this namespace and HtmlString is used commonly

# How to test
- Try building the website using the build/build.ps1 powershell script
- Install the program using the console
- Use the generated Templates to create a website
- See how to replicate the bug from here in the in issue thread
